### DISCUSSION TITLE
General Product Feedback

### DISCUSSION THREAD

**Discussion by michellemerrill** [community/community#4085] [Other Feature Feedback, Questions, & Ideas]
#### **Introducing our newest category addition - `General Feedback`**

📣 We've opened up space here for your suggestions on how our products could be improved and discuss those improvements with the community, including members of the GitHub product team. 

We are hoping to foster productive and collaborative conversations, so please check out [the README](https://github.com/github/feedback#readme) for guidance on kicking off a successful discussion!

**Comment by Levi-Lesches** [comment]
Just a reminder to people coming from @isaacs's GitHub feedback repo that instead of just adding 👍 to a topic, you should upvote the discussion as well (upper left corner)

**Reply by fayasnoushad** [reply]
Super 😘

**Reply by michellemerrill** [reply]
Thanks @Levi-Lesches ✨ Great reminder!

**Comment by huehnerlady** [comment]
This is amazing, waiting for that for a long time, thank you soooo much ❤️

**Comment by qinyihao** [comment]
I wanna ask how long can I get access to codespaces and copilot.

**Reply by michellemerrill** [reply]
@qinyihao Thanks for reaching out about GitHub Copilot.  You can sign up for the wait list at https://github.com/features/copilot/signup. We are expanding capacity every day and will be bringing people in gradually.  If you have any questions related to GitHub Copilot before joining the technical preview then please check out our FAQs at https://copilot.github.com/.

**Reply by qinyihao** [reply]
> @qinyihao Thanks for reaching out about GitHub Copilot. You can sign up for the wait list at https://github.com/features/copilot/signup. We are expanding capacity every day and will be bringing people in gradually. If you have any questions related to GitHub Copilot before joining the technical preview then please check out our FAQs at https://copilot.github.com/.

Thanks for your reply, @michellemerrill , but I'd like to know why I hadn't got access to it after approx 72 hrs while my friends got the access within 24 hrs. I'll appreciate it if you can answer this question for me.

**Reply by KoolenDasheppi** [reply]
> > @qinyihao Thanks for reaching out about GitHub Copilot. You can sign up for the wait list at https://github.com/features/copilot/signup. We are expanding capacity every day and will be bringing people in gradually. If you have any questions related to GitHub Copilot before joining the technical preview then please check out our FAQs at https://copilot.github.com/.
> 
> Thanks for your reply, @michellemerrill , but I'd like to know why I hadn't got access to it after approx 72 hrs while my friends got the access within 24 hrs. I'll appreciate it if you can answer this question for me.

I'm having a similar issue. I signed up the day it was announced and I've yet to be accepted in, yet I see so many others getting in almost instantly, especially in comparison to how long I've been waiting.
